packages/nextra/src/loader.ts

```diff
@@ -172,15 +176,16 @@ async function loader(
       defaultShowCopyCode,
       staticImage,
       flexsearch,
-      latex
+      latex,
+      codeHighlight
     },
     mdxPath,
-    true
+    false // TODO: produce hydration errors or error - Create a new processor first, by calling it: use `processor()` instead of `processor`.
```

Comment:
caching compiler is not ready for the moment, it produces hydration error on the client or an error
```bash
Create a new processor first, by calling it: use `processor()` instead of `processor`
```
while trying call `compiler.data('headingMeta', { headings: [] })` 2nd time